.github/workflows/gamemaker_build.yml

```
@@ -94,15 +94,15 @@ jobs:
       # This step sets up the GameMaker build CLI tool Igor https://github.com/bscotch/igor-setup
       - id: igor_setup
         name: Setup Igor
-        uses: bscotch/igor-setup@v1.1.0
+        uses: bscotch/igor-setup@v1.1.3
         with:
           target-yyp: ${{ steps.find_yyp.outputs.yyp-path }}
           access-key: ${{ secrets.GM_ACCESS_KEY }}
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)